proto/minder/v1/minder.proto

```diff
@@ -1438,6 +1498,11 @@ message Profile {
     // name is the name of the profile instance.
     string name = 3;
 
+    // labels are a set of system-provided attributes which can be used to
+    // filter profiles and status results.  Labels cannot be set by the user,
+    // but are returned in ListProfiles.
+    repeated string labels = 12;
```

Comment:
Are we gonna have labels be a fixed list of strings? or are we going to have labels be a keypair like k8s? This is something I'd like to get consensus on.

Reply:
Added details on the allowed strings.

Reply:
For the record, we've agreed that an array of labels was the way to go and we'd have a DNS-style restricted character set to begin with. Thanks @evankanderson for driving this!